backend/src/requests/user-content-request/user-content-request.service.ts

```diff
@@ -112,12 +114,18 @@ export class UserContentRequestService {
    * */
   async getAnswersOfQuestion(id: string, sortCriteria: QueryParameters) {
     const question = await this.userContentService.getQuestion(id);
-    if (question == null)
+    if (null === question || null === question.userContent)
```

Comment:
I am not fine with this order of the null check. But thats to be discussed.

Reply:
Why not remove the word null altogether and just write
`if (!question || !question.userContent)`

Reply:
Yes I know that that works, but someone in our team wanted the syntax with the `null` == `question`.

Reply:
In a short discussion we decided to move this to a general discussion and accept it in this way at this stage.